reactjs-todo/client/index.js

```diff
@@ -21,16 +21,18 @@ import { AppContext, useGlobalStateMgmt } from './global-state';
  */
 import './styles/index.scss';
 
-Config.set({
-  clientId: WEB_OAUTH_CLIENT,
-  redirectUri: `${APP_URL}/callback`,
-  scope: 'openid profile email',
+const urlParams = new URLSearchParams(window.location.search);
+const journeyParam = urlParams.get('journey');
+
+await Config.setAsync({
+  clientId: WEB_OAUTH_CLIENT, // e.g. PingOne Services Client GUID
+  redirectUri: `${window.location.origin}/callback`, // Redirect back to your app, e.g. 'https://localhost:8443/login?centralLogin=true' or the domain your app is served.
+  scope: "openid profile email address phone", // e.g. 'openid profile email address phone'
   serverConfig: {
-    baseUrl: AM_URL,
-    timeout: '5000',
+    wellknown: WELLKNOWN_URL,
+    timeout: '3000', // Any value between 3000 to 5000 is good, this impacts the redirect time to login. Change that according to your needs.
```

Comment:
I think timeout should be a number technically, right?

Reply:
it works like this. TBH not changed that to anything in terms of type so I think it's OK to leave like this. On Main we have it as such

Reply:
Yeah, it should be a number, but JS will coerce the string to a number, if needed.

Reply:
Actually, `timeout` isn't really needed anymore as it's totally optional. Even for Central Login, we don't rely on timeouts anymore if the user is not authenticated.